http-source/pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>http-source</artifactId>
+	<packaging>jar</packaging>
+	<name>Http Source</name>
```

Comment:
Wow, did not realize `${name}` in application.yml was actually maven substitution (was amazed that boot capitalized stuff, but did not look further). Thought `name` was contributed to the env by boot itself.
We *really* need to change the token to _e.g._ `@` make this more obvious.

Will remove in the meantime